src/ripple/app/ledger/LedgerTiming.cpp

```diff
@@ -128,7 +128,7 @@ bool ContinuousLedgerTiming::haveConsensus (
     }
 
     // If 80% of current proposers (plus us) agree on a set, we have consensus
-    if (((currentAgree * 100 + 100) / (currentProposers + 1)) > 80)
+    if (((currentAgree * 100 + 100) / (currentProposers + 1)) >= CONSENSUS_PCT)
```

Comment:
1.  We repeat this same logic twice.  Could I get you extract it out as a function?  This is a critical piece of our code - getting this right is one of our guards against forking the ledger! - so it's key to emphasize it.

2. The fact that all of these calculations are done in integer arithmetic is a little dodgy.  Experience shows that expressions using integer arithmetic have an order of magnitude more bugs in them than floating point ones do, because there are more gotchas.

I believe this expression to be correct as it is - but I'd be happier with a floating point expression for all critical decision-making code like this one.